.github/workflows/build_release.yml

```
@@ -0,0 +1,77 @@
+        required: true
+
+jobs:
+  server:
+    name: Build and release server
+    runs-on: ubuntu-latest
+    if: inputs.name || inputs.new_tag
+    env:
+      ARTIFACTS: server/dist/reearth_*.*
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)